README.md

```diff
@@ -1,19 +1,18 @@
 ## Build v2
 
-*Proposal / *Work-in-progress
+*Proposal*/*Work-in-progress*
 
-An API to build images on Kubernetes using popular strategies and tools like source-to-image,
-buildpack-v3, kaniko and buildah, in an extensible way.
+An API to build container-images on Kubernetes using popular strategies and tools like
+`source-to-image`, `buildpack-v3`, `kaniko` and `buildah`, in an extensible way.
 
 ## How
 
 The following are the `BuildStrategies` supported by this operator, out-of-the-box:
 
-* [Source-to-Image](samples/buildstrategy/buildstrategy_source-to-image_cr.yaml);
-* [Buildpacks-v3](samples/buildstrategy/buildstrategy_buildpacksv3-cr.yaml);
-* [Buildah](samples/buildstrategy/buildstrategy_buildah_cr.yaml);
-* [Kaniko](samples/buildstrategy/buildstrategy_kaniko_cr.yaml);
-
+* [Source-to-Image](samples/buildstrategy/source-to-image/README.md);
```

Comment:
I would like to show people the yaml right away :) Hence, wouldn't prefer pointing them to the readme.

Reply:
but wondering if it would be overwhelming? 🤔

Reply:
I think we should cover the build-strategy with specific documentation, so users can understand what that's about without having to go straight away, is indeed, less overwhelming than pure yaml.